### PR TITLE
Add the fusion of quantized batchnorm and relu

### PR DESCRIPTION
**Pull request by lly-zero-one**
As title, we want to support the BN2d_relu and BN3d_relu

Test to be added!

### CODE REVIEW

**Comment by supriyar** [comment]
Do we not need changes in the python side to do the fusion of these ops as well?

**Comment by lly-zero-one** [comment]
> Do we not need changes in the python side to do the fusion of these ops as well?

Yes, I have another PR in my sandbox. It will be together with conv3d and the quantization flow change.

**Comment by supriyar** [comment]
Can we merge this PR with the one that has the test for it? I'm wary of enabling fused Relu without testing it.

**Comment by lly-zero-one** [comment]
> Can we merge this PR with the one that has the test for it? I'm wary of enabling fused Relu without testing it.

I am adding a bn3drelu test to this PR.

File: test/test_quantized.py [hunk 1]
```diff
@@ -1394,6 +1394,42 @@ def test_batch_norm(self, X, Y_scale, Y_zero_point, qengine):
             quantize_ref = torch.quantize_per_tensor(float_ref, Y_scale, Y_zero_point, dtype_x)
             self.assertEqual(qy.int_repr().numpy(), quantize_ref.int_repr().numpy())
 
+    @given(X=hu.tensor(shapes=hu.array_shapes(min_dims=4, max_dims=5,
+                                              min_side=1, max_side=32),
+                       qparams=hu.qparams()),
+           Y_scale=st.floats(0.2, 2.6),
+           Y_zero_point=st.integers(0, 5),
+           qengine=st.sampled_from(("qnnpack", "fbgemm")))
```
Comment (supriyar) [inline test/test_quantized.py]: I believe there is no qengine for this op. We don't have qnnpack implementation.

Comment (lly-zero-one) [reply test/test_quantized.py]: ok, let's remove it.

**Review by supriyar — APPROVED** [review]
LGTM

**Comment by mruberry** [comment]
Unlanding. This appears to have broken the the Mac build:

```
Mar 19 16:57:02 Generating XML reports...
Mar 19 16:57:02 Running test_quantized ... [2020-03-19 16:57:02.241228]
Mar 19 16:57:03 
Mar 19 16:57:03 Running tests...
Mar 19 16:57:03 ----------------------------------------------------------------------
Mar 19 16:58:15 s.ssssssssss............libc++abi.dylib: terminating with uncaught exception of type c10::Error: quantized engine FBGEMM is not supported (setQEngine at ../aten/src/ATen/Context.cpp:113)
Mar 19 16:58:15 frame #0: c10::Error::Error(c10::SourceLocation, std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char> > const&) + 135 (0x10c3b0ee7 in libc10.dylib)
Mar 19 16:58:15 frame #1: at::Context::setQEngine(c10::QEngine) + 487 (0x11f7e8dc7 in libtorch_cpu.dylib)
Mar 19 16:58:15 frame #2: THPModule_setQEngine(_object*, _object*) + 75 (0x11ec2b72b in libtorch_python.dylib)
Mar 19 16:58:15 <omitting python frames>
Mar 19 16:58:15 
Mar 19 16:58:15 Traceback (most recent call last):
Mar 19 16:58:15   File "test/run_test.py", line 675, in <module>
Mar 19 16:58:15     main()
Mar 19 16:58:15   File "test/run_test.py", line 668, in main
Mar 19 16:58:15     raise RuntimeError(message)
Mar 19 16:58:15 RuntimeError: test_quantized failed! Received signal: SIGIOT
Mar 19 16:58:15 + cleanup
Mar 19 16:58:15 + retcode=1
Mar 19 16:58:15 + set +x
```

**Pull request closed without merging**